src/backend/tfjs_backend_test.ts

```diff
@@ -164,31 +164,31 @@ describeMathCPU('countParams', () => {
   it('Scalar', () => {
     const x = zeros([]);
     expect(K.countParams(x)).toEqual(1);
-    expect(K.countParams(new LayerVariable(x))).toEqual(1);
+    expect(K.countParams(new LayerVariable(x).read())).toEqual(1);
```

Comment:
LayerVariable didn't explicitly implement TensorInterface, but it did implicitly.

So option 1) as shown call .read() in these tests.
option 2) add LayerVariable to the type unions of Tensor|SymbolicTensor for the pure shape/dtype functions.

(If we take option 2, we probably want a export type TensorLike = Tensor|SymbolicTensor|LayerVariable; )